### PR TITLE
feat: prepare to open source.

### PR DESCRIPTION
**Pull request by benoitdevos**
logion-network/logion-internal#136

### CODE REVIEW

File: .env.logion [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Connect to a local node.
+#
+# For more information on how to use .env files and environment variables
+# consult the Configuration section in the README.
+
+SAS_SUBSTRATE_WS_URL=ws://127.0.0.1:9944
+NODE_TLS_REJECT_UNAUTHORIZED=0
+SAS_EXPRESS_PORT=8081
```
Comment (gdethier) [inline .env.logion:8]: Add `SAS_SUBSTRATE_TYPES` here as well (instead of setting it in `package.json` script entry)?

File: package.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "name": "@substrate/api-sidecar",
   "description": "REST service that makes it easy to interact with blockchain nodes built using Substrate's FRAME framework.",
   "homepage": "https://github.com/paritytech/substrate-api-sidecar#readme",
-  "author": "Parity Technologies <admin@parity.io>",
+  "author": "Logion Team",
```
Comment (gdethier) [inline package.json]: I would turn this into "contributors" (see [here](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#people-fields-author-contributors)) and mention both Parity and us.

**Review by gdethier — CHANGES_REQUESTED** [review]
Good job! A few remarks to address before merging...

File: .env.logion (hunk 1 above)
Comment (benoitdevos) [reply .env.logion:8]: I tried but failed - the env variable `$PWD` is not evaluated. If you have some trick to workaround this...

File: package.json (hunk 2 above)
Comment (benoitdevos) [reply package.json]: Done.

File: .env.logion (hunk 1 above)
Comment (gdethier) [reply .env.logion:8]: Nothing obvious that I can think about. Leave unchanged then, it's fine.

**Review by gdethier — APPROVED** [review]

**Pull request merged**